<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QWidget" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emittance measurement</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="QGroupBox">
     <property name="title">
      <string>Monitors</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="TableView" name="mtab"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox">
     <property name="title">
      <string>Fit results</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="TableView" name="rtab"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="button_update_monitor">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="std_buttons">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Apply|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="long_transfer">
          <property name="text">
           <string>Match from start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_dispersion">
          <property name="text">
           <string>Dispersion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="respect_coupling">
          <property name="text">
           <string>Coupled</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>madgui/widget/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
